lib/ads/interstitial_ad_manager.js

```diff
@@ -203,7 +203,7 @@ shaka.ads.InterstitialAdManager = class {
 
 
   /**
-   * @param {shaka.extern.Interstitial} interstitial
+   * @param {shaka.extern.HLSInterstitial} interstitial
    */
   async addMetadata(interstitial) {
```

Comment:
Now that the extern is called `HLSInterstitial`, perhaps this variable could be called `hlsInterstitial` to further differentiate it from the `AdInterstitial` objects being created below, and to make it clearer what kind of interstitial this method takes.

Reply:
Changed!